modules/fbs-core/api/src/main/scala/de/thm/ii/fbs/controller/ExportController.scala

```diff
@@ -22,8 +28,11 @@ class ExportController {
   private val taskExportService: TaskExportService = null
   @Autowired
   private val courseRegistrationService: CourseRegistrationService = null
+  @Autowired
+  private val taskService: TaskService = null
+  private val logger = LoggerFactory.getLogger(this.getClass)
 
-  @GetMapping(value = Array("/tasks/{taskId}/export"))
+  @GetMapping(value = Array("/tasks/{taskId}/export"), produces = Array(MediaType.APPLICATION_JSON_VALUE))
```

Comment:
```suggestion
  @GetMapping(value = Array("/tasks/{taskId}/export"))
```

This configuration is not necessary.